docs/pages/guides/CssOverrides.example.jsx

```diff
@@ -2,8 +2,7 @@
 import React, { useState } from 'react';
 import lightBlue from '@material-ui/core/colors/lightBlue';
 import { DatePicker } from '@material-ui/pickers';
-import { createMuiTheme } from '@material-ui/core';
-import { ThemeProvider } from '@material-ui/styles';
+import { createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles';
```

Comment:
We have removed `MuiThemeProvider` from the documentation.

```suggestion
import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
```

Reply:
oh god :)